src/components/card/CardSidebarTabDetails.vue

```diff
@@ -176,6 +177,23 @@ export default {
 				stringify: this.stringify,
 				parse: this.parse,
 			},
+			shortcuts: [
+				{
+					text: 'Today',
+					onClick() {
+						const date = new Date()
+						return date
```

Comment:
I'm not sure what today should ideally do but setting it to the current time will cause the file to be over due immediately.